erdpy/cli_contracts.py

```diff
@@ -359,3 +370,28 @@ def _send_or_simulate(tx: Transaction, contract: SmartContract, args: Any):
     output_builder = cli_shared.send_or_simulate(tx, args, dump_output=False)
     output_builder.set_contract_address(contract.address)
     utils.dump_out_json(output_builder.build(), outfile=args.outfile)
+
+
+def reproduce_build(args: Any):
```

Comment:
Optionally, can be renamed to `do_reproducible_build()`.

Reply:
Renamed it!